packages/appium/docs/en/quickstart/test-rb.md

```diff
@@ -3,7 +3,7 @@ title: Write a Test (Ruby)
 ---
 
 
-The [AppiumLib](https://github.com/appium/ruby_lib) and the [AppiumLibCore](https://github.com/appium/ruby_lib_core) are Appium officcial client libraries in Ruby, which are available via gem under the [appium_lib](https://rubygems.org/gems/appium_lib) and the [appium_lib_core](https://rubygems.org/gems/appium_lib_core) package names. The appium_lib_core inherits from the Selenium Ruby Binding, and the appium_lib inherits from the appium_lib_core, so intalling these libraries include the selenium binding.
+The [AppiumLib](https://github.com/appium/ruby_lib) and the [AppiumLibCore](https://github.com/appium/ruby_lib_core) (**recommended**) are Appium official client libraries in Ruby, which are available via gem under the [appium_lib](https://rubygems.org/gems/appium_lib) and the [appium_lib_core](https://rubygems.org/gems/appium_lib_core) package names. The appium_lib_core inherits from the Selenium Ruby Binding, and the appium_lib inherits from the appium_lib_core, so installing these libraries include the selenium binding. We recommend `appium_lib_core` if you needed less complexity in the client library. The `appium_lib` has some useful methods the core does not have, but it also has code complexity by historical reasons.
```

Comment:
> The `appium_lib` has some useful methods the core does not have, but it also has code complexity by historical reasons.

 The `appium_lib` has some useful methods the core does not have, but for the cost of greater complexity.